app/src/main/java/org/dhis2/data/dhislogic/DhisMapUtils.kt

```diff
@@ -9,6 +9,8 @@ import org.hisp.dhis.android.core.dataelement.DataElement
 import org.hisp.dhis.android.core.enrollment.Enrollment
 import org.hisp.dhis.android.core.event.Event
 import org.hisp.dhis.android.core.program.ProgramStage
+import org.hisp.dhis.android.core.trackedentity.TrackedEntityAttribute
+import org.hisp.dhis.android.core.trackedentity.TrackedEntityInstance
 
 class DhisMapUtils @Inject constructor(val d2: D2) {
```

Comment:
I know this for the SDK :+1: and is temporary (you don't need to remove or change). 

GetCoordinateAttributeInfo is an actual use-case from domain, I tend to avoid util classes since they represent a bit of "container" with many methods and tend to break Single responsability and open/close principle